phoenix-core/src/main/java/org/apache/phoenix/query/ConnectionQueryServicesImpl.java

```diff
@@ -1420,7 +1420,7 @@ public MetaDataMutationResult createTable(final List<Mutation> tableMetaData, fi
         byte[] tenantIdBytes = rowKeyMetadata[PhoenixDatabaseMetaData.TENANT_ID_INDEX];
         byte[] schemaBytes = rowKeyMetadata[PhoenixDatabaseMetaData.SCHEMA_NAME_INDEX];
         byte[] tableBytes = rowKeyMetadata[PhoenixDatabaseMetaData.TABLE_NAME_INDEX];
-        byte[] tableName = physicalTableName != null ? physicalTableName : SchemaUtil.getTableNameAsBytes(schemaBytes, tableBytes);
+        byte[] tableName = SchemaUtil.getTableNameAsBytes(schemaBytes, tableBytes);
```

Comment:
I think this is correct because this variable is used as the first argument of ensureTableCreated method and it doesn't expect a physical table name but a phoenix table name.
Also,  this variable is used by the following code but it expects a phoenix table name as well.

```
            ensureViewIndexTableCreated(
                    SchemaUtil.getPhysicalHBaseTableName(tableName, isNamespaceMapped, tableType).getBytes(),
                    tableProps, familiesPlusDefault, MetaDataUtil.isSalted(m, kvBuilder, ptr) ? splits : null,
                            MetaDataUtil.getClientTimeStamp(m), isNamespaceMapped);
```